### ISSUE TITLE
[Global] Accent color combination low contrast

### ISSUE THREAD

**Issue by svinkle** [Shopify/dawn#1236]
## Issue summary

In Sense, "Accent 1" and "Accent 2" generate instances of low text contrast.

Accent 1 text:
- Text: `#dfd0cb`
- Background: `#905b58`
- Contrast ratio:`3.66:1`

Accent 2 heading text:
- Text: `#faf7f3`
- Background: `#c0918f`
- Contrast ratio:`2.55:1`

Accent 2 text:
- Text: `#ebddda`
- Background: `#c29492`
- Contrast ratio:`1.99:1`

Accent 2 CTA:
- Text: `#a96b68`
- Background: `#faf7f3`
- Contrast ratio:`3.94:1`

<details>
  <summary>Screenshots</summary>

  <img width="422" alt="" src="https://user-images.githubusercontent.com/1392632/150375999-58e783e2-7aca-4ac5-87e7-12d7d6a073cf.png">
  <img width="436" alt="" src="https://user-images.githubusercontent.com/1392632/150376014-b58fdfa3-494e-4eda-9d0b-1969bf2cbe97.png">

</details>

<!-- ### Current code

#### HTML

```html

``` -->

## Steps to reproduce

1. Go to a [Sense theme demo store](https://theme-sense-demo.myshopify.com/) with any supported browser.
2. Check the [color contrast of the Accent 1 text](https://contrast-ratio.com/#%23dfd0cb-on-%23905b58).
3. Check the [color contrast of the Accent 2 heading text](https://contrast-ratio.com/#%23faf7f3-on-%23c0918f).
4. Check the [color contrast of the Accent 2 text](https://contrast-ratio.com/#%23ebddda-on-%23c29492).
5. Check the [color contrast of the Accent 2 CTA](https://contrast-ratio.com/#%23a96b68-on-%23faf7f3).

## Behavior

### Expected

- Color contrast to meet AA requirements.

### Actual

- Color contrast does not meet AA requirements.

## Recommendation

Adjust the value of the text/background color to increase the color contrast.

- Contrast ratio of `4.5:1` for normal size text (below `18` point or `14` point bold.)
- Contrast ratio of `3:1` for large text (above `18` point or `14` point bold.)

<!-- ### Recommended code

#### HTML

```html

``` -->

<!-- #### Diff

```diff
- line removed
+ line added
``` -->

## Specifications

- **Browser:** Any
- **Dawn version:** 2.5.0
- **Component:** Global
- **WCAG Principal:** Perceivable
- **WCAG SC:** [1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
- **Severity:** High
- **Effort:** Medium
